README.md

```diff
@@ -1,32 +1,190 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create a new database
+- Create a new table
+- Drop a table
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+The SQL `CREATE TABLE` statement creates a new table
+by specifying the table name and the column details, including
+column name and data type.    The `DROP TABLE` statement deletes a table
+from the database.
 
-## Topic 1
+## Create A New Database (Code-along)
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+We will create a new database to store information about pets and their owners.
 
-## Topic 2
+1. Launch the **pgAdmin** tool.
+2. Enter the password for the **postgres** user.
+3. You should see a default server **PostgreSQL 15** along with a database named **postgres**:   
+  
+![pgadmin default server](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/pgadmin.png)
+    
+4. Right-click on the `Databases` icon and select Create > Database:      
+  
+![new database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/newdb.png)
+    
+5. Name the new databases "pets" and press Save.    
+     
+![name pet database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/namepetsdb.png)
+    
+6. Confirm the new database named `pets` appears in the server view:    
+     
+![new pet database](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/newpetdb.png)
+
+
+We will use the **Query Tool** to execute SQL statements.
+
+1. Click on the `pets` database to select the database.
+2. Click on the `query tool` icon.
+ 
+![open query tool](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/openquerytool.png)
+
+The query tool opens a panel where we can write SQL statements.
+Confirm the connection is for  "pets/postgres@PostgreSQL 15". 
+
+![query tool](https://curriculum-content.s3.amazonaws.com/6002/sql-create-statement/querytool.png)
+
+
+
+## CREATE TABLE
+
+The `CREATE TABLE` statement creates a new table
+by specifying the table name and the column details.
+Each column is assigned a name and data type, along with other optional attributes.
+
+While PostgreSQL supports dozens of data types, we will primarily work with the following:
+
+- INTEGER stores integer values
+- DECIMAL stores fractional numeric values
+- TEXT stores textual values
+- DATE stores year, month, and day
+
+#### Coding Conventions
+
+1. Use uppercase for SQL keywords such as CREATE, TABLE, INTEGER, PRIMARY KEY, etc.
+   SQL is case insensitive, but it is a convention
+   to uppercase SQL keywords to separate them from table names and columns.
+2. Use lowercase letters for table and column names such as `pet`, `species`, etc.
+3. When we have multiple words in a table or column name,
+   link them together using underscores rather than spaces or camelcase.
```

Comment:
Should we also open the door for the debate of singular vs plural table names here too? Or at least mention consistency across table names?

Reply:
ok, added a brief comment about this.